src/patternfly/components/Slider/slider.scss

```diff
@@ -1,87 +1,90 @@
+  --#{$slider}__rail-track--before--fill--BackgroundColor: var(--pf-t--global--border--color--hover);
+  --#{$slider}__rail-track--before--BorderRadius: var(--pf-t--global--border--radius--tiny);
   --#{$slider}__rail-track--before--fill--BackgroundColor--gradient-stop: var(--#{$slider}--value);
-
-  @include pf-v5-bidirectional-style(
-    $prop: --#{$slider}__rail-track--before--fill--direction, 
-    $ltr-val: right,
-    $rtl-val: left
-  );
-
+    
   // steps
-  --#{$slider}__steps--FontSize: var(--#{$pf-global}--FontSize--sm);
+  --#{$slider}__steps--FontSize: var(--pf-t--global--font--size--sm);
   --#{$slider}__steps--Height: var(--#{$slider}__steps--FontSize);
-
+  
   // step tick
-  --#{$slider}__step-tick--Top: var(--#{$pf-global}--spacer--md);
+  --#{$slider}__step-tick--Top: var(--pf-t--global--spacer--md);
   --#{$slider}__step-tick--Width: #{pf-size-prem(4px)};
   --#{$slider}__step-tick--Height: #{pf-size-prem(4px)};
-  --#{$slider}__step-tick--BackgroundColor: var(--#{$pf-global}--BorderColor--200);
+  --#{$slider}__step-tick--BackgroundColor: var(--pf-t--global--icon--color--subtle);
   --#{$slider}__step-tick--TranslateX: -50%;
-  --#{$slider}__step-tick--BorderRadius: var(--#{$pf-global}--BorderRadius--lg);
-  --#{$slider}__step--m-active__slider-tick--BackgroundColor: var(--#{$pf-global}--primary-color--100);
+  --#{$slider}__step-tick--BorderRadius: var(--pf-t--global--border--radius--large);
+  --#{$slider}__step--m-active__slider-tick--BackgroundColor: var(--pf-t--global--icon--color--brand--clicked);
   --#{$slider}__step--first-child__step-tick--TranslateX: 0;
   --#{$slider}__step--last-child__step-tick--TranslateX: -100%;
-
+  
   // step label
   --#{$slider}__step-label--TranslateX: -50%;
-  --#{$slider}__step-label--Top: calc(var(--#{$pf-global}--spacer--xl) + var(--#{$slider}__rail-track--Height));
+  --#{$slider}__step-label--Top: calc(var(--pf-t--global--spacer--xl) + var(--#{$slider}__rail-track--Height));
   --#{$slider}__step--first-child__step-label--TranslateX: 0;
   --#{$slider}__step--last-child__step-label--TranslateX: -100%;
-
+  
   // thumb
-  --#{$slider}__thumb--Top: calc(var(--#{$slider}__rail-track--Height) / 2 + var(--#{$pf-global}--spacer--md));
+  --#{$slider}__thumb--Top: calc(var(--#{$slider}__rail-track--Height) / 2 + var(--pf-t--global--spacer--md));
   --#{$slider}__thumb--Width: #{pf-size-prem(16px)};
   --#{$slider}__thumb--Height: #{pf-size-prem(16px)};
   --#{$slider}__thumb--Left: var(--#{$slider}--value);
-  --#{$slider}__thumb--BackgroundColor: var(--#{$pf-global}--primary-color--100);
+  --#{$slider}__thumb--BackgroundColor: var(--pf-t--global--color--brand--default);
   --#{$slider}__thumb--TranslateX: -50%;
   --#{$slider}__thumb--TranslateY: -50%;
-  --#{$slider}__thumb--BorderRadius: var(--#{$pf-global}--BorderRadius--lg);
+  --#{$slider}__thumb--BorderRadius: var(--pf-t--global--border--radius--large);
```

Comment:
I think a circle border radius would be useful - cc @lboehling. The value would be "50%" and it will make any square a circle, regardless of size.